festipals/src/components/SignUpForm.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+export default class SignUpForm extends Component {
+
+  render() {
+    return (
+      <div>
+      <div className="Container-fluid">
+      <div className="row">
+          <div className="col-xs-12">
+            <PageHeader icon="fa fa-info" title="Sign Up" description="(Sign up for FestiPals)"/>
```

Comment:
Not sure if we need the PageHeader here either, but we can discuss this tomorrow.